pykern/pkio.py

```diff
@@ -18,11 +18,26 @@
 import re
 import shutil
 import six
+import tempfile
 
 #: used during unit testing see ``pykern.pkunit.save_chdir``
 pkunit_prefix = None
 
 
+def atomic_write(filename, content, mode='w'):
+    """Writes to file atomically
```

Comment:
Need text that is distinct from the name, e.g. Overwrites an existing file via rename to ensure integrity.